src/components/TheDetails.vue

```diff
@@ -615,8 +628,36 @@ export default {
 			'toggleAllDay',
 			'moveTask',
 			'setClassification',
+			'getTaskByUri',
 		]),
 
+		async loadTask() {
+			if (this.task === undefined || this.task === null) {
+				const calendars = this.calendar ? [this.calendar] : this.calendars
+				for (const calendar of calendars) {
+					this.loading = true
+					try {
+						const found = await this.getTaskByUri({ calendar: calendar, taskUri: this.$route.params.taskId })
+							.then((response) => {
```

Comment:
please don't mix try..;catch and .then/catch :)

Reply:
What would be the correct way? Could you elaborate a bit, I am not really familiar with this.

Reply:
It's usually good practice to not use .then/.catch anymore as you end up with chaining lots of promises :)
Using async/await is recommended ;)

Reply:
Ok, I will have a look. Thanks!

Reply:
Thanks for making me aware @skjnldsv. Looks much cleaner with async/await only 👍